gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.groovy

```diff
@@ -32,12 +33,19 @@ class BaselineErrorProne extends AbstractBaselinePlugin {
             errorprone "com.palantir.baseline:baseline-error-prone:${extractVersionString()}"
         }
 
-        project.tasks.withType(JavaCompile) {
-            options.compilerArgs += [
-                    "-XepDisableWarningsInGeneratedCode",
-                    "-Xep:EqualsHashCode:ERROR",
-                    "-Xep:EqualsIncompatibleType:ERROR",
-            ]
+        project.afterEvaluate { p ->
```

Comment:
according to error-prone docs you need to configure bootclasspath with java8 since error-prone shades some of the jdk (sun.misc) dependencies with java9 versions

Reply:
Unfortunately calling `getAsPath` on a configuration will force evaluation and we need to delay it. There's no api in JavaCompile nor Test task to set it lazily